packages/server/test/unit/plugins/child/run_plugins_spec.js

```diff
@@ -1,433 +1,278 @@
 require('../../../spec_helper')
```

Comment:
All these tests had been skipped for over a year, so I unskipped them, fixed the existing ones, and added new ones for my changes to `run_plugins.js`. I doesn't have 100% test coverage of `run_plugins.js` (I didn't add tests for any functionality I didn't add or change), but it's a step in the right direction.